openassessment/xblock/test/test_staff.py

```diff
@@ -283,25 +306,37 @@ def test_staff_assess_handler(self, xblock):
         self.submit_staff_assessment(xblock, submission, assessment=TEAM_GOOD_ASSESSMENT)
 
         assessment = teams_api.get_latest_staff_assessment(submission['team_submission_uuid'])
-        self._assert_team_assessment(assessment, submission)
+        self._assert_team_assessment(assessment, submission, self.expected_answer)
+
+    @scenario('data/team_submission.xml', user_id='Bob')
+    def test_staff_assess_handler_regrade(self, xblock):
+        submission = self._setup_xblock_and_create_team_submission(xblock)
+        submission["uuid"] = str(submission["submission_uuids"][0])
```

Comment:
maybe get the submission from the api or construct a new dict? kinda misleading to pass the team submission after just changing the uuid

Reply:
Not quite sure what you mean. Can you explain?

Reply:
`submission` here uis the team submission, right? that's what's returned from `_setup_xblock_and_create_team_submission()`? and it looks like you're just adding in a 'uuid' key with one of the individual submission uuids

Reply:
Ah! The reason I do that is because xblock/test/test_staff.py sets 'submission_uuid':
`assessment['submission_uuid'] = submission['uuid']`. This uuid is not used in asserts. Without doing this, I would get a KeyError. This is a submission_uuid for one of the team members, not the team.